### PR TITLE
Enable version upgrades by dependabot

### PR DESCRIPTION
**Pull request by karthikbtf**
Lets see how this works before we can apply it on terraware-web as well

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (constanzauanini) [inline .github/dependabot.yml:6]: Show we do this weekly?

Comment (karthikbtf) [reply .github/dependabot.yml:6]: was thinking we can see sooner how this works.. but yes, can get spammy. will switch to weekly

Comment (karthikbtf) [reply .github/dependabot.yml]: Updated this..

**Comment by karthikbtf** [comment]
I'm going to merge this in so we can see something when we get back next week.

**Review by constanzauanini — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (constanzauanini) [reply .github/dependabot.yml]: Thanks!

**Pull request merged**